.travis.yml

```diff
@@ -34,9 +34,11 @@ matrix:
 install:
 - rvm use 2.3.1 --install --binary --fuzzy
 - bundle install --jobs=3 --retry=2 --path=vendor/bundle
-- if ! shellcheck --version ; then
-    curl -sSL "${SHELLCHECK_URL}" | tar -C "${HOME}/bin" -xjf - ;
+- if ! shellcheck --version &>/dev/null; then
+    curl -sSL "${SHELLCHECK_URL}"
+    | tar --exclude 'SHA256SUMS' --strip-components=1 -C "${HOME}/bin" -xjf -;
```

Comment:
Well, the previous tarball didn't have a directory at the top level, which was mostly just me being sloppy.  The new tarball has a directory that includes a binary and a `SHA256SUMS` file, which is arguably more in keeping with tradition.